app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -1,6 +1,11 @@
 class ManageIQ::Providers::Vmware::CloudManager::RefreshParser < ManageIQ::Providers::CloudManager::RefreshParser
   include ManageIQ::Providers::Vmware::RefreshHelperMethods
 
+  # While parsing the VMWare catalog only those vapp templates whose status
+  # is reported to be "8" are ready to be used. The documentation says this
+  # status is POWERED_OFF, however the cloud director shows it as "Ready"
+  VAPP_TEMPLATE_STATUS_READY = "8".freeze
```

Comment:
I missed it while I was updating the code based on your suggestions :(. Amended commit has this fixed.